src/main/java/com/healthsync/ui/PatientScene.java

```diff
@@ -1,31 +1,426 @@
+        ptDOBBox.setAlignment(Pos.BASELINE_RIGHT);
+
+        //Phone Field
+        Label Phone = new Label("Phone Number");
+        Phone.setStyle("-fx-font-size: 12px; -fx-font-weight: bold");
+        PatientInformation.getChildren().add(Phone);
+        String ptphone = patient.getContactInformation().toString().substring(0,9);
+        Label ptPhone = new Label(ptphone);
+        ptPhone.setStyle("-fx-font-size: 11px");
+        HBox ptPhoneBox = new HBox(ptPhone);
+        PatientInformation.getChildren().add(ptPhoneBox);
+        ptPhoneBox.setPadding(new Insets(0,0,10,0));
+        ptPhoneBox.setAlignment(Pos.BASELINE_RIGHT);
+
+        //Email Field
+        Label Email = new Label("Email Address");
+        Email.setStyle("-fx-font-size: 12px; -fx-font-weight: bold");
+        PatientInformation.getChildren().add(Email);
+        String ptemail = patient.getContactInformation().substring(11,(patient.getContactInformation().toString().length()));
+        Label ptEmail = new Label(ptemail);
+        ptEmail.setStyle("-fx-font-size: 11px");
+        HBox ptEmailBox = new HBox(ptEmail);
+        PatientInformation.getChildren().add(ptEmailBox);
+        ptEmailBox.setPadding(new Insets(0,0,10,0));
+        ptEmailBox.setAlignment(Pos.BASELINE_RIGHT);
+
+        //Insurance Field
+        Label Insurance = new Label("Insurance");
+        Insurance.setStyle("-fx-font-size: 12px; -fx-font-weight: bold");
+        PatientInformation.getChildren().add(Insurance);
+        String ptinsurance = patient.getInsuranceInformation().substring(0,(patient.getInsuranceInformation().indexOf(',')))
+                + "\n" + patient.getInsuranceInformation().substring((patient.getInsuranceInformation().indexOf(',')+1),patient.getInsuranceInformation().length());
+        Label ptInsurance = new Label(ptinsurance);
+        ptInsurance.setStyle("-fx-font-size: 11px");
+        HBox ptInsuranceBox = new HBox(ptInsurance);
+        PatientInformation.getChildren().add(ptInsuranceBox);
+        ptInsuranceBox.setPadding(new Insets(0,0,10,0));
+        ptInsuranceBox.setAlignment(Pos.BASELINE_RIGHT);
+
+        //Pharmacy Field
+        Label Pharmacy = new Label("Pharmacy");
+        Pharmacy.setStyle("-fx-font-size: 12px; -fx-font-weight: bold");
+        PatientInformation.getChildren().add(Pharmacy);
+        String ptpharm = patient.getPharmacyInformation().substring(0,(patient.getPharmacyInformation().indexOf(',')))
+                + "\n" + patient.getPharmacyInformation().substring((patient.getPharmacyInformation().indexOf(',')+1),patient.getPharmacyInformation().length());
+        Label ptPharmacy = new Label(ptpharm);
+        ptPharmacy.setStyle("-fx-font-size: 11px");
+        HBox ptPharmacyBox = new HBox(ptPharmacy);
+        PatientInformation.getChildren().add(ptPharmacyBox);
+        ptPharmacyBox.setPadding(new Insets(0,0,10,0));
+        ptPharmacyBox.setAlignment(Pos.BASELINE_RIGHT);
+
+//        StackPane EditInfo = new StackPane();
```

Comment:
Whats this commented out section for?

Reply:
That was going to be the Edit Patient Info function, but I couldn't finish the functionality for updating the database with the new info, so I scrapped that feature altogether. I don't think that part was in the requirements though.